process/block/sovereignChainBlock.go

```diff
@@ -181,9 +187,29 @@ func (scbp *sovereignChainBlockProcessor) CreateBlock(initialHdr data.HeaderHand
 	}
 
 	if scbp.epochStartTrigger.IsEpochStart() {
+		err := scbp.createBlockStarted()
```

Comment:
You can move this at line 188 and deleted at line 228

Reply:
Right, fixed it